package/src/const/bridges.ts

```diff
@@ -1,5 +1,6 @@
 export enum BridgeType {
   ibc = 'ibc',
+  ics20 = 'ics20',
```

Comment:
new bridge type, because message flow is a bit different and not easy to decide. Could also be done with type "ibc" and then registering the tokens somewhere, as ics20. Still feels better to distinct here.